_posts/2023-02-19-a-lean-dataops-platform.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+inject_optimize_tag: true
+title: "A Lean Modern Data Platform"
+description: ""
+date: 2023-02-22
+highlight: monokai
+author: Matthew Carter
+tags: [snowflake, data-ops, data platform, fivetran, dbt, dagster]
+---
+
+# Building a Lean ‘Modern’ Data Platform
+## Introduction
+Right now, everyone wants to be data driven and there has been an explosion of tooling to help people do that. However, to become data driven with things like dashboards, reporting, AI and ML, it is paramount that these soon-to-be groan inducing buzzwords are built upon a solid foundation of quality, aggregated and easy to access data. A well engineered data platform is, in my opinion, the answer to this problem. For companies looking to effectively aggregate and analyze large amounts of data, building a platform is essential. *Excel sheets can only get you so far.* With a data platform, businesses generally can gain insights into their operations, customer behavior, and market trends. This knowledge can help them make informed decisions and ultimately move closer to becoming data driven. 
```

Comment:
`A well engineered data platform is, in my opinion, the answer to this problem` Which problem? It might be worth to make the problem statement a bit more clear

Reply:
completed 🫡